code/__DEFINES/mobs.dm

```diff
@@ -7,6 +7,9 @@
 #define ORGAN_DEAD       8
 #define ORGAN_MUTATED    16
 
+// /obj/item/bodypart on_mob_life() retval flag
+#define BODYPART_LIFE_UPDATE_HEALTH (1<<0)
```

Comment:
just make it 1. 1<<0 = 1 1<<1 = 2.
Since we don't use bitshift logic to define other bit related things.